bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -382,6 +382,24 @@ class AudioManager {
     return bridge.exitAudio();
   }
 
+  forceExitAudio() {
+    this.isConnected = false;
+    this.isConnecting = false;
+    this.isHangingUp = false;
+
+    if (this.inputStream) {
+      this.inputStream.getTracks().forEach((track) => track.stop());
+      this.inputStream = null;
+      this.inputDevice = { id: 'default' };
+    }
+
+    window.parent.postMessage({ response: 'notInAudio' }, '*');
+    window.removeEventListener('audioPlayFailed', this.handlePlayElementFailed);
+
+    const bridge = (this.useKurento && this.isListenOnly) ? this.listenOnlyBridge : this.bridge;
```

Comment:
Not necessarily related with this pull request but I was checking this out over this manager and we use it multiple times. We should move it to a single place.